capture_service/device_mgr.cc

```diff
@@ -97,6 +100,31 @@ absl::Status IsAppInstalled(const AdbSession &adb, std::string_view package)
     return adb.Run(absl::StrFormat("shell pm path %s", package));
 }
 
+absl::Status InstallVulkanLayer(const AdbSession &adb,
+                                std::string_view  package,
+                                std::string_view  layer_filename)
+{
+    RETURN_IF_ERROR(
+    adb.Run(absl::StrFormat(R"(push "%s" "%s")",
+                            ResolveAndroidLibPath(std::string(layer_filename)).generic_string(),
+                            kTargetPath)));
+    RETURN_IF_ERROR(adb.Run(
+    absl::StrFormat(R"(shell run-as %s cp "%s/%s" .)", package, kTargetPath, layer_filename)));
+    return absl::OkStatus();
+}
+
+absl::Status UninstallVulkanLayer(const AdbSession &adb,
```

Comment:
@angela28chen  do you think should we also call this in the device cleanup, so that it has all the cleanup stuff.